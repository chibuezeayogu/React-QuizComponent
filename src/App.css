.error {
  color: red;
  text-align: center;
}

.container,
.question, 
.button-container, 
.quiz-end, 
.question, 
.answers-container {
  display: flex;
  flex-direction: column;
}

.container {
  background: #F0F3F4;
  height: 600px;
  width: 80%;
  position: relative;
  border: 2px solid gray;
  margin: 0 auto;
  top: 100px;
}

.question, .button-container {
  height: 20%;
  width: 95%;
  padding: 0px;
  margin: 10px auto;
  font-family: sans-serif;
  align-items: center;
  justify-content: center;
}

.question {
  background: #FFFFFF;
  font-size: 24px;
}

.answers-container {
  background: #FFFFFF;
  height: 50%;
  width: 95%;
  padding: 0px;
  margin: 10px auto;
  display: flex;
  justify-content: flex-start;
}

.button {
  height: 15%;
  width: 95%;
  justify-content: flex-start;
  font-size: 16px;
  margin: 10px auto;
  padding: 10px;
  border: 1px solid black;
}

.button:hover {
  background: #ecf7f9;
  color: black;
}

.quiz-end {
  align-items: center;
  height: 25%;
  width: 70%;
  margin: 0 auto;
  border: 1px solid rgb(40, 110, 16);
  padding: 10px;
  background-color: aliceblue;
  top: 200px;
  position: relative;
}

button {
  height: 50px;
  width: 100px;
  border: 1px solid rgb(40, 110, 16);
}
