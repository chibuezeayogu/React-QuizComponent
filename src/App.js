import React from 'react';

import quizData from './quiz_data.json';
import './App.css'
import Quiz from './Quiz'

const App = () => (
  <Quiz quizData={quizData} />
)

export default App 