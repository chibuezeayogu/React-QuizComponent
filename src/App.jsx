import React from 'react';

import quizData from './quizData.json';
import './App.css'
import Quiz from './Quiz'

const App = () => (
  <Quiz quizData={quizData} />
)

export default App 