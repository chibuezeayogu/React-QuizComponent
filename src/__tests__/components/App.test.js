import React from 'react';
import { shallow } from 'enzyme';

import App from '../../App';

describe('QuizEnd Component', () => {
  const wrapper = shallow(<App />);

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
});