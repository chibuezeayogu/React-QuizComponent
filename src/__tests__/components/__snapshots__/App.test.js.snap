// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizEnd Component should render correctly 1`] = `
<Quiz
  quizData={
    Object {
      "quizQuestions": Array [
        Object {
          "answer": "7",
          "answerOptions": Array [
            "5",
            "6",
            "7",
            "8",
          ],
          "id": 1,
          "instructionText": "How many continents are there on Planet Earth?",
        },
        Object {
          "answer": "4",
          "answerOptions": Array [
            "1",
            "2",
            "3",
            "4",
          ],
          "id": 2,
          "instructionText": "What's your favorite number?",
        },
      ],
    }
  }
/>
`;
